utilities/virt.py

```
@@ -2111,22 +2112,59 @@ def vm_instance_from_template(
 
 
 @contextmanager
-def node_mgmt_console(admin_client, node, node_mgmt):
+def cordon_node(admin_client, node):
+    """Cordon a node and uncordon it on exit."""
     hco_namespace = get_hco_namespace(admin_client=admin_client)
     try:
-        LOGGER.info(f"{node_mgmt.capitalize()} the node {node.name}")
-        extra_opts = "--delete-emptydir-data --ignore-daemonsets=true --force" if node_mgmt == "drain" else ""
+        LOGGER.info(f"Cordon the node {node.name}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)